src/doctopus/web.clj

```diff
@@ -53,7 +53,7 @@
 (def substitutions {"$URL_ROOT"  (:ip (server-config))})
 
 ;; Let's start bootstrapping!
```

Comment:
Arg, yes it is -- and the github UI didn't refresh until after I hit merge. Will take that out in the search branch.